### PR TITLE
Turn it up to 11!

### PR DESCRIPTION
**Pull request by thomasvandoren**
Bump version number to 1.11.0. Copies what was done in 769c350.

This release is one louder. Most releases only go to 10. They're on 10, all the way
up, all the way up. Where can they go from there? No where.

Not this release. It goes to 11. It's one louder. We needed that little push over the
cliff, so we put it up to 11.

image

### CODE REVIEW

File: GOALS [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ============================================
-Chapel Public Release Goals (version 1.10.0)
+Chapel Public Release Goals (version 1.11.0)
 ============================================
 
 TODO: Update these goals.
```
Comment (bradcray) [inline GOALS]: Yeesh, was this in there in the last release?  Let's remove it.  These goals are good enough for now.

**Comment by bradcray** [comment]
The previous time we changed this, commit 59b1ee5 updated the following .bad files, some of which were claimed to be due to version numbers.  Ideally, the various filtering steps that Tom and Vass have implemented will have rendered these unnecessary to change manually this time around, but could you spot-check if you haven't?

M       test/functions/iterators/tzakian/par_iter_recursive.bad
M       test/parallel/begin/sungeun/autoCopyWithBegin.bad
M       test/parallel/begin/sungeun/initCopyWithBegin.bad
M       test/users/vass/div-ceil-on-more-int-types.bad

(and if changes are required, maybe grep to see if there are others, or extend the filtering to handle these cases?)

**Comment by thomasvandoren** [comment]
The .bad filtering has improved such that we no longer need to update those .bad files. I ran `git grep '\.10\.'` and then start_test -futures on the tests that had version numbers in them. They all passed.

**Comment by bradcray** [comment]
Cool, looks good to me, then!

**Pull request merged**